src/graphics/aurora/highlightabletext.h

```diff
@@ -32,7 +32,8 @@ class HighlightableText: public Text, public Highlightable {
 
   public:
 	HighlightableText(const FontHandle &font, const Common::UString &str,
-	     float r = 1.0f, float g = 1.0f, float b = 1.0f, float a = 1.0f, float halign = kHAlignLeft);
+	     float r = 1.0f, float g = 1.0f, float b = 1.0f, float a = 1.0f,
+	     float halign = kHAlignLeft, float valign = kVAlignTop);
```

Comment:
This isn't really visible on GitHub here, but this commit, which uses `kVAlignTop` comes before the 98d5bfd3c74b4205fc723716152fd67003ae3955, which defines `kVAlignTop` in the first place.

Which means that when going back to this commit, for example during a git bisect, this commit here won't compile. I'd like to avoid commits not compiling, if possible, even if later commits fix the error.

You should probably swap these two commits.